wherescape/connectors/hubspot/process_data.py

```diff
@@ -6,6 +6,8 @@
 This module processes the collected data so it can be send to the Hubspot Module
 """
 
+IGNORED_COLUMNS = []
```

Comment:
issue: `IGNORED_COLUMNS` is not used in the module

Reply:
oh yes. I forgot to remove it after choosing a different approach. I will remove it now